core/src/main/java/org/infinispan/interceptors/MarshalledValueInterceptor.java

```diff
@@ -258,7 +258,9 @@ public Object visitReplaceCommand(InvocationContext ctx, ReplaceCommand command)
       return new AbstractDelegatingEntryCacheSet<K, V>(getCacheWithFlags(cache, command), set) {
          @Override
          public CloseableIterator<CacheEntry<K, V>> iterator() {
-            return new CloseableIteratorMapper<>(super.iterator(), e -> unwrapEntry(e, ctx));
+            // We pass a null ctx, since this is always local invocation - if it was remote it would use
+            // DistributionBulkInterceptor
```

Comment:
Doesn't this mean that reusing the `ctx` parameter would also be ok, because it's always local?